trigger/database/database.go

```diff
@@ -169,6 +184,8 @@ func (t *TriggerRunner) cockroachTrigger(ctx context.Context, conn *crdb.Client,
 			columns = append(columns, string(field.Name))
 		}
 
+		rowsArray := make([]map[string]any, 0)
+		groupTriggerKey := ""
```

Comment:
Do we have a specific use for the key? If we don't I think we should just pass an empty key to start with.

Reply:
I was only concerned if the key is used internally in some other place in locutus, and it prevents any re-deployment if the same key is used. So constructed a key that represents the result set.

Reply:
With database triggers I don't think it makes a difference as they are run every X seconds anyway.

Reply:
Ok, in that case, will go with an empty key.